script.js

```diff
@@ -1,28 +1,90 @@
-var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K", "A"];
+var values  = [2, 3, 4, 5, 6, 7, 8, 9, 10, "Jack", "Queen", "King", "Ace"];
 var suits   = ["Clubs", "Diamonds", "Hearts", "Spades"];
 var game = {
   deck: [],
   players: [],
   hands: [],
   buildDeck: function(){
-
+    for (var i=0; i<values.length;i++){
+      for (var x=0;x<suits.length; x++){
+        var card = {
+          value: values[i],
+          suit: suits[x],
+          trueDetective: i
+        }
+        this.deck.push(card);
+      }
+    }
+    return this.deck;
   },
   shuffleDeck: function(){
-
+    for (var i=0; i<52;i++) {
```

Comment:
Try not to use "magic numbers" -- I'd use `cards.length` or `deck.length` or something instead.